content/en/docs/user-documentation/how-tos/offline-containers.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Offline/disconnected container images
+draft: false
+weight: 9
+tags:
+  - Container Storage
+  - Disconnected
+description: Offline containers are containers which are stored in the operating system image, and made available to `cri-o` via the `/etc/container/storage.conf` `additionalimagestores` list.
+---
+## What are offline container images
+
+Offline containers are containers which are stored in the operating system image,
+and made available to `cri-o` via the `/etc/container/storage.conf` `additionalimagestores`
+list.
+
+Those container images are accesible for `cri-o` to create containers. Those images
+cannot be deleted, but newer versions of those containers can be downloaded normally,
+which `cri-o` will store in the general R/W container storage of the system.
+
+## When to use offline container images
+
+Offline containers are useful when the Edge device will have restricted connectivity,
+or no connectivity at all. Those containers are also helpful to improve general MicroShift
+and application startup on first boot, since no images need to be downloaded from the
+network and the applications are readily available to `cri-o`
+
+## RPM packaging of container images
+
+RPM packaging of container images into read-only container storage is offered via the
+[`paack`](https://github.com/redhat-et/microshift/blob/main/packaging/rpm/paack.py) tool
+as an experimental method to allow users to create ostree images containing the desired containers.
+`RPM` was not designed for storing files with numeric uids/gids, or containing extended attributes,
+although several workarounds allow this we are looking for better ways to provide this.
+
+## Offline MicroShift containers images
+
+MicroShift uses a set of containers for the minimal components which can be installed
+on the operating system image, those are published [here](https://copr.fedorainfracloud.org/coprs/g/redhat-et/microshift-containers/), and can also be manually built using:  `packaging/rpm/make-microshift-images-rpm.sh`.
+
+To install the MicroShift container images you can use:
+```bash
+curl -L -o /etc/yum.repos.d/microshift-containers.repo \
+          https://copr.fedorainfracloud.org/coprs/g/redhat-et/microshift-containers/repo/fedora-35/group_redhat-et-microshift-containers-fedora-35.repo
+
+rpm-ostree install microshift-containers
+````
+
+Or simply install that repository and package from an os build blueprint.
```

Comment:
Do you mean to install the `microshift-containers` RPM in an rpm-ostree base OS image using something like Image Builder?  or do you mean OSbuild as a tool? 

I'd make this one sentence a bit more clear.